src/main/java/com/conveyal/datatools/manager/models/Project.java

```diff
@@ -45,6 +45,9 @@
     /** Last successful auto deploy. **/
     public Date lastAutoDeploy;
 
+    /** The most recently entered Pelias webhook URL. Used when creating new Deployments */
+    public String lastUsedPeliasWebhookUrl;
```

Comment:
I don't think this provides much value and recommend removing it.

Reply:
Removed in 7aab4a64041e70370cce800d09b930135cadc2d2 and 3a6c67521105b3b5a190409d25888a332231eda9